app/utility/base_planning_svc.py

```diff
@@ -163,7 +163,7 @@ def _is_fact_bound(fact):
     @staticmethod
     async def _exclude_existing(link, operation):
         all_hostnames = [agent.host for agent in await operation._active_agents()]
-        for item in link.relationships:
```

Comment:
@brianedmonds90 can you confirm this?

Reply:
From testing, it would appear that link.relationships is never populated anymore. Most code seems to have replaced it with stuff related to link.requirements

Reply:
Link.relationships is still populated by the parsers. Link.used is populated by the planning_svc. Would it be a better approach to prevent lateral movement by checking that 'remote.host' is not equal to any existing values stored in agent.host for each agent in the operation?

Reply:
^^ oh wait I think I read this wrong